### ISSUE TITLE
Implement Witches’ brew  poisoning attack 

### ISSUE THREAD

**Issue by Nathalie-B** [Trusted-AI/adversarial-robustness-toolbox#1386]
**Is your feature request related to a problem? Please describe.**
Implement poisoning attack: 
`Witches' Brew: Industrial Scale Data Poisoning via Gradient Matching` 
https://arxiv.org/abs/2009.02276 

**Describe the solution you'd like**

- Jupyter notebook illustrating the use of the attack 
- Example folder 
- Implementation for Pytorch and Keras framework.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by TS-Lee** [comment]
I will work on this.
